AIMS/Views/Home/Index.cshtml

```diff
@@ -1,85 +1,133 @@
+        new Dictionary<string, string> { {"Asset Name", "HP Pavillion TP01-2234"}, {"Type", "Desktop"}, {"Tag #", "DT-0075"}, {"Assigned To", "Unassigned"}, {"Status", "Available"} },
+        new Dictionary<string, string> { {"Asset Name", "Samsung Galaxy Book4"}, {"Type", "Laptop"}, {"Tag #", "LT-0005"}, {"Assigned To", "Caitlin Clark (90334)"}, {"Status", "Damaged"} },
+        new Dictionary<string, string> { {"Asset Name", "Logitech Zone Vibe 100"}, {"Type", "Headset"}, {"Tag #", "HS-0015"}, {"Assigned To", "Brian Regan (27094)"}, {"Status", "In Repair"} },
+        new Dictionary<string, string> { {"Asset Name", "Belkin BoostCharge 3.3ft USB-C"}, {"Type", "Charging Cable"}, {"Tag #", "CC-0088"}, {"Assigned To", "Unassigned"}, {"Status", "Available"} },
+        new Dictionary<string, string> { {"Asset Name", "Dell Inspiron 3030"}, {"Type", "Desktop"}, {"Tag #", "DT-0100"}, {"Assigned To", "Maximillian Brandt"}, {"Status", "Assigned"} },
+        new Dictionary<string, string> { {"Asset Name", "Poly Voyager 4320"}, {"Type", "Headset"}, {"Tag #", "HS-0001"}, {"Assigned To", "Emily Carter (47283)"}, {"Status", "In Repair"} },
+        new Dictionary<string, string> { {"Asset Name", "j5create 100W Super Charger"}, {"Type", "Charging Cable"}, {"Tag #", "CC-0019"}, {"Assigned To", "Bruce Wayne (34532)"}, {"Status", "Damaged"} },
+        new Dictionary<string, string> { {"Asset Name", "Dell Inspiron 15"}, {"Type", "Laptop"}, {"Tag #", "LT-0115"}, {"Assigned To", "Kate Rosenberg (93232)"}, {"Status", "Assigned"} }
     };
-} 
+}
 
-<!-- Placeholder for Summary Cards -->
-<div class="summary-cards-placeholder"
+<!-- Original Layout - Summary Cards Placeholder -->
+<div class="summary-cards-placeholder">
     <p>Summary Cards Placeholder - Replace this with summary cards section</p>
 </div>
 
-<!-- Asset Table Section -->
+<!-- Original Layout - Asset Table Section -->
 <div class="asset-management">
     <div class="asset-header">
         <div class="left-section">
+            <!-- Search Container (modified for functionality) -->
             <div class="search-container">
-                <span class="input-group-text search-icon" id="search-addon" onclick="triggerSearch()">🔍</span>
-                <input type="text" class="form-control search-input" placeholder="Search for assets or employees..."
-                    aria-label="Search" aria-describedby="search-addon">
+                <span class="search-icon">🔍</span>
+                <input type="text" id="searchInput" 
+                       class="search-input" 
+                       placeholder="Search for assets or employees...">
             </div>
-            <button class="view-full-list-button">View Full List | 📋 | ≡</button>
+            <button id="resetSearch" class="view-full-list-button">View Full List | 📋 | ≡</button>

```

Comment:
Just curious why this specific button has this reset functionality assigned to it?